hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -1286,7 +1282,7 @@ public Config setDistributedClassloadingConfig(DistributedClassloadingConfig dis
         return this;
     }
 
-    private <T> T lookupByPattern(Map<String, T> configPatterns, String itemName) {
+    <T> T lookupByPattern(Map<String, T> configPatterns, String itemName) {
```

Comment:
It's used in the `com.hazelcast.config.ConfigCompatibilityChecker#checkCompatibleConfigs`. Perhaps we can do this differently, haven't thought about it.